packages/tools/viewer/src/viewer.ts

```diff
@@ -1367,20 +1384,17 @@ export class Viewer implements IDisposable {
         return true;
     }
 
+    private _shouldModelRenderPredicate(model: Model) {
+        return model._shouldRender();
+    }

```

Comment:
I think it is actually ok for this to just be an inline lambda. I did some quick memory profiling and it doesn't look like the lambdas on line 1060 actually result in any memory allocations. I'm guessing the JS engine is smart enough to see that this lambda doesn't need to be re-allocated since it doesn't capture any local state.